src/cli/fake_data/mod.rs

```diff
@@ -186,6 +195,197 @@ fn register_functions(tera: &mut Tera) {
     tera.register_function("userName", |_: &HashMap<String, Value>| {
         Ok(Value::from(Username(EN).fake::<String>()))
     });
+
+    // Group "number"
+    tera.register_function("digit", |_: &HashMap<String, Value>| {
+        Ok(Value::from(Digit(EN).fake::<String>()))
+    });
+    tera.register_function("numberWithFormat", |args: &HashMap<String, Value>| {
+        let format = match args.get("format") {
+            Some(val) => match from_value::<String>(val.clone()) {
+                Ok(v) => v,
+                Err(_) => {
+                    return Err(tera::Error::msg(format!(
+                        "Function `numberWithFormat` received format={} but `format` can only be a string",
+                        val
+                    )));
+                }
+            },
+            None => String::from("#"),
+        };
+        let format_str = Box::leak(format.into_boxed_str());
```

Comment:
I think we should add a comment saying that we know this "leaks" memory into the running program, but it should hopefully be called once and it is tiny. But since it's technically still a leak...